src/chrome/content/rules/STA-Travel.com.xml

```diff
@@ -0,0 +1,75 @@
+<!--
+	Other related rulesets:
+		- STA-Travel.at.xml
+		- STA-Travel.ch.xml
+		- STA-Travel.de.xml
+		- STA-Travel.co.jp.xml
+		- STA-Travel.co.nz.xml
+		- STA-Travel.co.th.xml
+		- STA-Travel.co.za.xml
+		- STA-Travel.com.au.xml
+		- STA-Travel.com.sg.xml
+
+
+	Non-functional subdomains:
+		- statravel.com		(t)
+			- articles	(t)
+			- links.edeals	(m)
+			- ibe	(t)
+			- owa	(no route to host)
+			- stacreativekit	(m)
+			- webchat	(t)
+			- webforms	(r)
+			- ww2	(no route to host)
+
+		- (www.)statravel.com.cn	(ssl connection error)
+			- en	(i)
+
+		- statravel.se		(t)
+			- www		(m)
+
+		- (www.)statravel.dk		(m)
+
+	e: expired certificate
+	h: http redirect
+	i: invalid certificate chain
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="STA Travel.com">
+
+	<target host="statravel.com" />
+	<target host="www.statravel.com" />
+	<target host="blog.statravel.com" />
+	<target host="book.statravel.com" />
+	<target host="car-rental.statravel.com" />
+	<target host="customercare.statravel.com" />
+	<target host="g.statravel.com" />
+	<target host="helios.statravel.com" />
+	<target host="hotels.statravel.com" />
+	<target host="login.statravel.com" />
+	<target host="m.statravel.com" />
+	<target host="marketing.statravel.com" />
+	<target host="my.statravel.com" />
+	<target host="api.my.statravel.com" />
+	<target host="api.partners.my.statravel.com" />
+	<target host="rail.statravel.com" />
+	<target host="tours.statravel.com" />
+		<test url="http://tours.statravel.com/content/images/icon-cal-small.png" />
+	<target host="usvpn.statravel.com" />
+
+	<securecookie host=".+" name=".+" />
+
+	<!-- http redirect -->
+	<exclusion pattern="^http://www\.statravel\.com/(blog|tours)" />
+		<test url="http://www.statravel.com/blogs/" />
+		<test url="http://www.statravel.com/tours-worldwide.htm" />
```

Comment:
Are there any other `tours*` or could you use a more restrictive regex?

Reply:
`/tours.htm` is also valid, redirects to `/tours-worldwide.htm`.